sink.patch

```diff
@@ -1,21 +1,22 @@
 diff --git a/pkg/sink/sink.go b/pkg/sink/sink.go
-index c6d63bfb..633b75a2 100644
+index c6d63bfb..990dcced 100644
 --- a/pkg/sink/sink.go
 +++ b/pkg/sink/sink.go
-@@ -24,6 +24,7 @@ import (
+@@ -24,6 +24,8 @@ import (
  	"fmt"
  	"io/ioutil"
  	"net/http"
++	"os"
 +	"strings"
  	"sync"
  
  	"github.com/tektoncd/triggers/pkg/apis/triggers"
-@@ -419,7 +420,7 @@ func (r Sink) ExecuteInterceptors(trInt []*triggersv1.TriggerInterceptor, in *ht
+@@ -419,7 +421,7 @@ func (r Sink) ExecuteInterceptors(trInt []*triggersv1.TriggerInterceptor, in *ht
  		}
  
  		// TODO: Plumb through context from EL
 -		interceptorResponse, err := interceptors.Execute(context.Background(), r.HTTPClient, &request, url.String())
-+		interceptorResponse, err := interceptors.Execute(context.Background(), r.HTTPClient, &request, strings.ReplaceAll(url.String(), ".tekton-pipelines.", ".kcpa9f18e6516b976c21e45eb38fd4291927a3c9dd86fda1b7b7c03ead1."))
++		interceptorResponse, err := interceptors.Execute(context.Background(), r.HTTPClient, &request, strings.ReplaceAll(url.String(), os.Getenv("SINK_KCP_NS"), "."+os.Getenv("SINK_PLNSVC_NS")+"."))
```

Comment:
Did this work? If so, we need to set the env variables on the other end.k

Reply:
Reverted.